srcDoc/command_line_parameters.rst

```diff
@@ -233,8 +233,8 @@ Test Parameters for Publishing and Subscribing Applications
    therefore setting them in the XML file has no effect; see the See
```

Comment:
line 171: is there a reason we have colons after each, latency, throughput, and announcement?

Reply:
I don't see any reason, I will change that.